snuba/web/split.py

```diff
@@ -216,8 +216,16 @@ def execute(
         from_date = util.parse_datetime(from_date_str, date_align)
 
         if from_date != from_date_ast:
+            logger.warning(
```

Comment:
This seems to be caused by some recent queries that include some timestamp conditions  beyond extensions that were not present before. The legacy representation would ignore such conditions, while the AST handles them. So the result may be different.